runtime/examples/memtest/memtest.cpp

```diff
@@ -0,0 +1,177 @@
+#include <array>
```

Comment:
Did you run clang-format on this code? It seems like it doesn't conform to our formatting guidelines (but might just be me looking at this in the browser).

Reply:
Are there special formatting rules used in TaPaSCo or just the standard clang-format style?

Reply:
Yes, there are .clang-format where necessary such as https://github.com/esa-tu-darmstadt/tapasco/blob/master/runtime/.clang-format . I usually use a simple combination of find and .clang-format, e.g. `find . -iname *.h -o -iname *.c -o -iname *.cpp -o -iname *.hpp | xargs clang-format -i -style=file -verbose`

Reply:
You can also use the Git integration of `clang-format`, e.g. `git clang-format HEAD~1`, so you don't have to use shell magic for finding the right files. 
See, for example: https://www.electronjs.org/docs/development/clang-format

Reply:
Even better in combination with filter-branch so we don't have any whitespace only commits:
https://stackoverflow.com/questions/43974371/run-git-clang-format-on-series-of-git-commits

Reply:
I tried to use filter-branch, but it does not seem to work. The output says it has modified the respective files and git also tells me afterwards that my local branch and the origin have diverged. But the files seem to be unchanged.
If I run `git-clang-format` manually it for example breaks some longer lines, when I use filter-branch to modify the commits the long lines are still there.
Do you have any idea what could be the problem?

Reply:
Maybe it needs some additional flags to use the correct clang-format File?

Reply:
I use the same flags so this should not be the problem

Reply:
And when I run `git-clang-format` manually without flags it also makes changes like breaking some lines

Reply:
Are the files that are broken part of your commit, or maybe some other files that haven't been properly formatted before?

Reply:
It only changes the two files of my commits

Reply:
But I can just include all clang-format changes in the commit with the comments. Then it's not a whitespace only commit